autoscaler/config/rbac/role.yaml

```diff
@@ -0,0 +1,155 @@
+---
```

Comment:
we actually remove this file in https://github.com/keyval-dev/odigos/pull/1080/files#diff-fb2e8a5552d2370c11c7fc2bc4017437b097f16f72ef5f37863e72d6f598aec8

there is something in the makefile that re-gernate it that we need to remove, as we do not actually use these yamls in odigos

Reply:
Right... removing it and `scheduler/config/rbac/role.yaml`, since I cannot discover a usecase for that either.